data/tilesets/wonders/guangzhou_port_facility.license

```diff
@@ -0,0 +1,2 @@
+SPDX-License-Identifier: GPL-3.0-or-later
+SPDX-FileCopyrightText: 1996-2021 Freeciv Contributors
```

Comment:
[first appearance in 2017 here](https://forum.freeciv.org/f/viewtopic.php?p=93762#p93762)